x/payment/client/testutil/integration_test.go

```diff
@@ -14,6 +14,8 @@ import (
 	cosmosnet "github.com/cosmos/cosmos-sdk/testutil/network"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
+	etypes "github.com/celestiaorg/celestia-app/x/payment/types"
```

Comment:
this is a small nit pick, but I think we should simply use types here instead of etypes as that is more consistent with the rest of the usage of types.

Reply:
I solved this